### PR TITLE
Asm migration

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
The content to be added to Changelog is as follows:
* ASM reserved ip migration commands implemented

### CODE REVIEW

File: lib/commands/asm/network.generated/network._js [hunk 1]
```diff
@@ -14,131 +14,228 @@
...
---name $p0
-
-azure vnet validate-migration 
---name $p0
-
-
-*/
+ azure vnet validate-migration
+ --name $p0
+ */
```
Comment (romangromov) [inline lib/commands/asm/network.generated/network._js]: Please update Generated Command List there

File: lib/commands/asm/network.generated/network._js [hunk 2]
```diff
@@ -14,131 +14,228 @@
...
```
Comment (EvgenyAgafonchikov) [inline lib/commands/asm/network.generated/network._js]: Please update the list with reserved-ip commands

File: lib/commands/asm/network.generated/network._js [hunk 3]
```diff
@@ -14,131 +14,228 @@
...
 
 'use strict';
 var profile = require('../../../util/profile');
 var utils = require('../../../util/utils');
 
 var $ = utils.getLocaleString;exports.init = function (cli) {
```
Comment (EvgenyAgafonchikov) [inline lib/commands/asm/network.generated/network._js]: Line is not wrapped

File: lib/commands/asm/network.generated/network._js (hunk 2 above)
Comment (MikhailTryakhov) [inline lib/commands/asm/network.generated/network._js]: Generated code is now in your task list :) Please go ahead with tests and comments

**Comment by MikhailTryakhov** [comment]
Hi @amarzavery can you please make a review of generated code and tests?

File: lib/commands/asm/network.generated/network._js [hunk 4]
```diff
@@ -14,131 +14,241 @@
...
+      if (!name) {
+        name = cli.interaction.promptIfNotGiven($('name : '), name, _);
+      }
+
+      cli.output.verbose('name = ' + name);
+      var subscription = profile.current.getSubscription(options.subscription);
+      var networkManagementClient = utils.createNetworkClient(subscription);
+      var result = networkManagementClient.networks.abortMigration(name, _);
+      if (result) {
+        cli.output.json(result);
```
Comment (amarzavery) [inline lib/commands/asm/network.generated/network._js]: Is this correct? Wouldnt you want to output things in a proper format and only output in json format if --json switch was provided.

File: lib/commands/asm/network.generated/network._js [hunk 5]
```diff
@@ -14,131 +14,241 @@
...
+      if (!name) {
+        name = cli.interaction.promptIfNotGiven($('name : '), name, _);
+      }
+
+      cli.output.verbose('name = ' + name);
+      var subscription = profile.current.getSubscription(options.subscription);
+      var networkManagementClient = utils.createNetworkClient(subscription);
+      var result = networkManagementClient.networks.commitMigration(name, _);
+      if (result) {
+        cli.output.json(result);
```
Comment (amarzavery) [inline lib/commands/asm/network.generated/network._js]: same as above

**Comment by MikhailTryakhov** [comment]
Hi @amarzavery 
issues fixed, PR is ready for merge

**Pull request merged**